### PR TITLE
Conditional

### PR DESCRIPTION
**Pull request by jessdtate**
some spelling fixes and link to new web docs

### CODE REVIEW

File: src/Interface/Application/ModuleWidget.cc [hunk 1]
```diff
@@ -564,6 +562,12 @@ void ModuleWidget::fillReplaceWithMenu()
     fullWidgetDisplay_->getModuleActionButton());
 }
 
+void ModuleWidget::menuFunction()
```
Comment (aylakhan) [inline src/Interface/Application/ModuleWidget.cc]: Empty function?

File: src/Interface/Application/ModuleWidget.cc [hunk 2]
```diff
@@ -732,7 +736,6 @@ void ModuleWidget::addOutputPortsToWidget(int index)
   }
   else
```
Comment (aylakhan) [inline src/Interface/Application/ModuleWidget.cc]: Empty else block?

File: src/Interface/Application/NetworkEditor.cc [hunk 3]
```diff
@@ -817,7 +833,6 @@ class NetworkSearchEngine
       }
       else
```
Comment (aylakhan) [inline src/Interface/Application/NetworkEditor.cc]: Empty else block?

**Comment by jessdtate** [comment]
Looks like I merged the subnets branch into this one, which I probably shouldn't have.  Should I wait until that branch is merged?

**Comment by dcwhite** [comment]
Yes. I'll try to get it merged today.

**Pull request merged**